rosbag2_keyboard_handler/src/IKeyboardHandler.cpp

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021 Apex.AI, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "rosbag2_keyboard_handler/IKeyboardHandler.hpp"
```

Comment:
I wouldn't expect that we would need a `.cpp` file for a pure-virtual interface.

Reply:
Yeah.. I also don't like to have a `.cpp` file for interface definition. But it's required for static constants definition. \
Another options might be to move those static constants to the `KeyboardHandlerBase` class but in this case when someone will  be looking to the interface it wouldn't be so obvious that some predefined constants exists which you can use.